internal/manifest/constants.go

```diff
@@ -3,32 +3,37 @@
 
 package manifest
 
-// TrustedMeasurement contains the expected launch digest and is injected at build time.
-var TrustedMeasurement = "000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000"
+import (
+	"encoding/json"
+	"fmt"
+	"os"
 
-// Default returns a default manifest.
-func Default() Manifest {
-	return Manifest{
-		ReferenceValues: ReferenceValues{
-			TrustedMeasurement: HexString(TrustedMeasurement),
-		},
+	"github.com/edgelesssys/contrast/node-installer/platforms"
+)
+
+// Default returns a default manifest with reference values for the given platform.
+func Default(platform platforms.Platform) *Manifest {
+	if embeddedReferenceValues == nil {
+		// If we're here, this is the first time this function is called, and the global state is not
+		// yet initialized. So let's unmarshal the embedded reference values. This does not need to be
+		// locked, as embeddedReferenceValues has a fixed value at compile time, making this idempotent.
```

Comment:
Could this be done in an `init()` function?

Reply:
I did this before, but @katexochen doesn't like init functions :D

Reply:
I mean, as far as I understand the data model assigning a pointer _should_ be safe, but still counts as a race condition. init would have the benefit of implicit synchronization.